src/main/java/com/theocean/fundering/domain/celebrity/controller/CelebController.java

```diff
@@ -0,0 +1,41 @@
+package com.theocean.fundering.domain.celebrity.controller;
+
+import com.theocean.fundering.domain.celebrity.dto.*;
+import com.theocean.fundering.domain.celebrity.service.CelebService;
+import com.theocean.fundering.global.utils.ApiUtils;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequiredArgsConstructor
+@RestController
+public class CelebController {
+    private final CelebService celebService;
+    @PostMapping("/celebs/register")
```

Comment:
```suggestion
    @PostMapping("/celebs")
```
이렇게 되어야 더 RESTful 할 것 같은데 어떨까요?

Reply:
네, 멘토님 해당 사항 반영하겠습니다!